ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -402,6 +403,27 @@ def test_ipa_healthcheck_dna_plugin_returns_warning_pagure_issue_60(self):
         for check in data:
             assert check["result"] == "SUCCESS"
 
+    def test_ipa_healthcheck_log_rotate_file_exist_issue35(self):
+        """
+        This test checks if log rotation has been added
+        for ipa-healthcheck tool so that logs are rotated
+        in /var/log/ipa/healthcheck folder.
+        This is a testcase for below pagure issue
+        https://github.com/freeipa/freeipa-healthcheck/issues/35
+        """
+        msg = "error: /etc/logrotate.d/ipahealthcheck:"
```

Comment:
You can use the constant HEALTHCHECK_LOG_ROTATE_CONF:
msg = "error: {}:".format(HEALTHCHECK_LOG_ROTATE_CONF)